OmiseSDK/Omise3DSViewController.swift

```diff
@@ -0,0 +1,129 @@
+        let navigationController = storyboard.instantiateViewController(withIdentifier: "Default3DSVerificationControllerWithNavigation") as! UINavigationController
+        let viewController = navigationController.topViewController as! Omise3DSViewController
+        viewController.authorizedURL = authorizedURL
+        viewController.delegate = delegate
+        
+        return navigationController
+    }
+    
+    /// A factory method for creating a 3DS verification view controller comes in UINavigationController stack.
+    ///
+    /// - parameter authorizedURL: The authorized URL given in `Charge` object that will be set to `Omise3DSViewController`
+    /// - parameter delegate: A delegate object that will recieved 3DS verification events.
+    ///
+    /// - returns: An `Omise3DSViewController` with given authorized URL and delegate.
+    public static func make3DSViewControllerWithAuthorizedURL(_ authorizedURL: URL, delegate: Omise3DSViewControllerDelegate) -> Omise3DSViewController {
+        let storyboard = UIStoryboard(name: "OmiseSDK", bundle: Bundle(for: Omise3DSViewController.self))
+        let viewController = storyboard.instantiateViewController(withIdentifier: "Default3DSVerificationController") as! Omise3DSViewController
+        viewController.authorizedURL = authorizedURL
+        viewController.delegate = delegate
+        
+        return viewController
+    }
+    
+    public override func loadView() {
+        super.loadView()
+        
+        webView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(webView)
+        
+        if #available(iOS 9.0, *) {
+            webView.topAnchor.constraint(equalTo: topLayoutGuide.bottomAnchor).isActive = true
+            webView.bottomAnchor.constraint(equalTo: bottomLayoutGuide.topAnchor).isActive = true
+            webView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+            webView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+        } else {
+            let views = ["webView": webView] as [String: UIView]
+            let constraints = NSLayoutConstraint.constraints(withVisualFormat: "|[webView]|", options: [], metrics: nil, views: views) +
+                NSLayoutConstraint.constraints(withVisualFormat: "V:|[webView]|", options: [], metrics: nil, views: views)
+            view.addConstraints(constraints)
+        }
+    }
+    
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+        webView.navigationDelegate = self
+    }
+    
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        start3DSProcess()
+    }
+    
+    private func start3DSProcess() {
+        guard let authorizedURL = authorizedURL else {
+            return
+        }
+        
+        let request = URLRequest(url: authorizedURL, cachePolicy: .reloadIgnoringLocalAndRemoteCacheData, timeoutInterval: 60.0)
```

Comment:
But the `URL` type of iOS API just validated that it's a valid URL but doesn't do any further validation like is it a file URL or `https` scheme. So if we need to validate more than that it's just a valid `URL` string then we need to do it by ourself